### PR TITLE
Add Devices with Trunks to vlan detail

### PR DESCRIPTION
**Pull request by edsonarios**
Issue link #1718
```
softlayer-python/ () $ slcli vlan detail -h
Usage: slcli vlan detail [OPTIONS] IDENTIFIER

        Get details about a VLAN.

┌────┬───────────────┬─────────────────────────────┐
│    │ identifier    │                             │
│    │ --no-vs       │ Hide virtual server listing │
│    │ --no-hardware │ Hide hardware listing       │
│    │ --no-trunks   │ Hide devices with trunks    │
│ -h │ --help        │ Show this message and exit. │
└────┴───────────────┴─────────────────────────────┘

softlayer-python/ () $ slcli vlan detail 1404269
┌──────────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────┐
│             name │ value                                                                                                │
├──────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────┤
│               id │ 1404269                                                                                              │
│           number │ 1632                                                                                                 │
│       datacenter │ Dallas 10                                                                                            │
│   primary_router │ bcr01a.dal10.softlayer.com                                                                           │
│ Gateway/Firewall │ -                                                                                                    │
│          subnets │ ┌─────────┬───────────────┬─────────────────┬───────────────┬────────────────────┬────────────┐      │
│                  │ │   id    │  identifier   │     netmask     │    gateway    │        type        │ usable ips │      │
│                  │ ├─────────┼───────────────┼─────────────────┼───────────────┼────────────────────┼────────────┤      │
│                  │ │ 2290190 │ 10.176.39.128 │ 255.255.255.192 │ 10.176.39.129 │ ADDITIONAL_PRIMARY │     61     │      │
│                  │ └─────────┴───────────────┴─────────────────┴───────────────┴────────────────────┴────────────┘      │
│               vs │ none                                                                                                 │
│         hardware │ none                                                                                                 │
│           trunks │ ┌──────────────────────────┬────────┬──────────────────────────────────────────────────────────────┐ │
│                  │ │          device          │  port  │                             tags                             │ │
│                  │ ├──────────────────────────┼────────┼──────────────────────────────────────────────────────────────┤ │
│                  │ │ host17.vmware.chechu.com │ eth0/2 │ tape tool: vmware-demo-lab-chechu, tape tool: chechu-testing │ │
│                  │ │ host16.vmware.chechu.com │ eth0/2 │ tape tool: vmware-demo-lab-chechu, tape tool: chechu-testing │ │
│                  │ │ host15.vmware.chechu.com │ eth0/2 │ tape tool: vmware-demo-lab-chechu, tape tool: chechu-testing │ │
│                  │ └──────────────────────────┴────────┴──────────────────────────────────────────────────────────────┘ │
└──────────────────┴──────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

### CODE REVIEW

File: SoftLayer/CLI/vlan/detail.py [hunk 1]
```diff
@@ -79,6 +88,21 @@ def cli(env, identifier, no_vs, no_hardware):
...
+            for trunk in trunks:
+                trunks_table.add_row([utils.lookup(trunk, 'networkComponent', 'downlinkComponent',
+                                                   'hardware', 'fullyQualifiedDomainName'),
+                                      utils.lookup(trunk, 'networkComponent', 'downlinkComponent',
+                                                   'networkComponentGroup', 'membersDescription'),
+                                      formatting.tags(utils.lookup(trunk, 'networkComponent', 'downlinkComponent',
+                                                                   'hardware', 'tagReferences'))])
+            table.add_row(['trunks', trunks_table])
+        else:
+            table.add_row(['trunks', 'none'])
```
Comment (caberos) [inline SoftLayer/CLI/vlan/detail.py:104]: please changes
```suggestion
            table.add_row(['trunks', ' - '])
```

File: SoftLayer/managers/network.py [hunk 2]
```diff
@@ -429,15 +429,22 @@ def get_subnet(self, subnet_id, **kwargs):
...
-        :param int id: The unique identifier for the VLAN
+        :param int vlan_id: The unique identifier for the VLAN
+        :param string mask: mask for request
         :returns: A dictionary containing a large amount of information about
                   the specified VLAN.
-
         """
-        return self.vlan.getObject(id=vlan_id, mask=DEFAULT_GET_VLAN_MASK)
+
+        _mask = None
```
Comment (caberos) [inline SoftLayer/managers/network.py:441]: I think remove this line

File: SoftLayer/CLI/vlan/detail.py (hunk 1 above)
Comment (edsonarios) [reply SoftLayer/CLI/vlan/detail.py]: Done

File: SoftLayer/managers/network.py (hunk 2 above)
Comment (edsonarios) [reply SoftLayer/managers/network.py]: Done

**Review by caberos — APPROVED** [review]
LGTM

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**